generator/tests/init-sources/init-sources.spec.ts

```diff
@@ -21,6 +21,9 @@ import { YargsMockup } from "../cdn.spec";
 
 describe("Test Extensions", () => {
 
+    // Increase timeout to 30 seconds
+    jest.setTimeout(30000);
```

Comment:
Mabe it's mine own opinion, but  I think you should not Increase timeout for all tests. You can set up longer timeout for one of them:

```
test("test init sources generator", async () => {
// blah-blah some test.
}, 30000);
```

Reply:
There are many tests which can failed with timeout